src/containers/BuildUnsignedSweepCoin/BuildUnsignedSweepCoin.tsx

```diff
@@ -224,7 +225,7 @@ function Form() {
                     maxPriorityFeePerGas: toWei(maxPriorityFeePerGas),
                   },
                   replayProtectionOptions: {
-                    chain: bitGoEnvironment === 'prod' ? 1 : 5,
+                    chain: coin === 'ethw' ? 10001 : bitGoEnvironment === 'prod' ? 1 : 5,
```

Comment:
can we move this to a function that returns the chain? e.g. `getChain(env)`

Reply:
also, are we only supporting ethw for prod?

Reply:
yep ethw is just supported for withdrawals in prod!